terraform/aks/workspace-variables/dv_review_aks.tfvars.json

```diff
@@ -25,5 +25,6 @@
       "replicas": 1,
       "memory_max": "1Gi"
     }
-  }
+  },
+  "deploy_temp_data_storage_account": false
```

Comment:
I think azure_tempdata_storage_account_name and azure_resource_group_name become irrelevant if deploy_temp_data_storage_account is false, but deploy-plan fails if you remove them.

Reply:
I just tested by removing those params and yes deploy-plan prompts you to fill them in so might be best to just keep them in?

Reply:
preference would be to not have to specify vars we don't need